main.go

```diff
@@ -357,3 +363,39 @@ func setupKcpWatcherReconciler(mgr ctrl.Manager, options controller.Options, fla
 		os.Exit(1)
 	}
 }
+
+func dropV1alpha1FromStoredVersions(mgr manager.Manager) {
```

Comment:
we'd better put it in a separate go routing to avoid blocking the main thread.

refactor this function to accept the version to be dropped, so that we can reuse it in the later deletion.